manifest.yml

```diff
@@ -1,9 +1,11 @@
 ---
 applications:
-    - name: opre-ops-test
-      buildpacks:
-        - python_buildpack
-
+  - name: opre-ops-test
+    buildpack: python_buildpack
+    # TODO: Django docs say not to use the `runserver` command in prod;
+    # switch over to a production-ready webserver like gunicorn
+    command: cd opre_ops && python manage.py migrate && python manage.py runserver 0.0.0.0:8080
 services:
  - opre-ops-psql-db
- 
\ No newline at end of file
+
```

Comment:
`OPRE_OPS_ENV_SERVICE` will now be part of the `services`

Reply:
Oooooh! And that will do the "bind" step automatically? Looking at:

https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#services-block

Reply:
Also should this be lowercase/kebab-case to match `opre-ops-psql-db`?

Reply:
Yes.  It looks like that's what they use, I am unsure if there's an actual style guide, but looking at [examples](https://docs.cloudfoundry.org/devguide/services/user-provided.html), it uses lowercase with hyphens

Reply:
Gotcha, thank you!  Will make those changes!